src/styles/blocks/card.scss

```diff
@@ -0,0 +1,106 @@
+body {
+  margin: 0;
+  box-sizing: border-box;
+  display: flex;
+  justify-content: center;
+  font-family: $font-family;
+}
+
+.card {
+  width: 198px;
+  height: 406px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px 40px;
+
+    background: url(../images/imac.jpeg) center/160px 134px no-repeat;
+  }
+
+  &__title {
+    display: flex;
+    width: 166px;
+    height: 36px;
```

Comment:
```suggestion

```
Consider using padding on the parent block instead of hardcoding the width. Also, the height of the element should depend on the content